### PR TITLE
Integrate parameter_sampling in retries tool

### PR DESCRIPTION
**Pull request by JohanWBorg**
- parameter_sampling can now sample on both normal and UCP scale.
- Retries tool using create_rng instead of random.random
- Added argument "rng" to AMD tool for random sampling (currently only for retries tool)

### CODE REVIEW

File: src/pharmpy/modeling/parameter_sampling.py [hunk 1]
```diff
@@ -85,6 +85,9 @@ def _sample_from_function(
     """
     rng = create_rng(rng)
 
+    parameter_estimates = parameter_estimates[
```
Comment (rikardn) [inline src/pharmpy/modeling/parameter_sampling.py:88]: There is a better way of doing this. Use `model.parameters.fixed` eller `model.parameters.nonfixed

File: src/pharmpy/modeling/parameter_sampling.py [hunk 2]
```diff
@@ -181,8 +184,12 @@ def sample_parameters_uniformly(
...
         samples = np.empty((n, len(lower)))
         for i, (x, a, b) in enumerate(zip(pe, lower, upper)):
-            lower = max(a, x - x * fraction)
-            upper = min(b, x + x * fraction)
+            if scale == 'normal':
+                lower = max(a, x - x * fraction)
+                upper = min(b, x + x * fraction)
+            elif scale == 'UCP':
+                lower = 0.1 - 0.1 * fraction
+                upper = 0.1 + 0.1 * fraction
```
Comment (rikardn) [inline src/pharmpy/modeling/parameter_sampling.py:192]: For sampling on UCP scale don't we still want the resulting parameter samples on the normal scale?

Comment (rikardn) [reply src/pharmpy/modeling/parameter_sampling.py:192]: Or perhaps it is faster to do conversion later.

File: docs/retries.rst [hunk 3]
```diff
@@ -55,6 +55,8 @@ The arguments of the retries tool are listed below.
 | prefix_name                                     | String determining prefix of model names such that models are named |
 |                                                 | {prefix_name}_retries_run2 for instance.                            |
 +-------------------------------------------------+---------------------------------------------------------------------+
+| rng                                             | A random number generator or seed to use for random sampling.
```
Comment (rikardn) [inline docs/retries.rst:58]: I see that we sometimes call this `rng` and sometimes `seed`. Actually we support either or. What do you think we should call it to be consistent?

**Review by rikardn — CHANGES_REQUESTED** [review]

File: src/pharmpy/modeling/parameter_sampling.py (hunk 1 above)
Comment (JohanWBorg) [reply src/pharmpy/modeling/parameter_sampling.py:88]: updated to use model.parameters.nonfixed instead

**Pull request merged**